controllers/chat/sendAnonymousMessage.js

```diff
@@ -5,6 +5,7 @@ const {MESSAGE_TYPE} = require('../../helpers/constants');
 const {determineMessageType, processReplyMessage} = require('../../utils/chat');
 const UsersFunction = require('../../databases/functions/users');
 const {User, UserBlockedUser} = require('../../databases/models');
+const BetterSocialCore = require('../../services/bettersocial');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `BetterSocialCore` module is imported but there's no check to ensure that it's properly initialized or available. This could lead to runtime errors if the module fails to initialize correctly. Consider adding error handling around this import.
```diff
+ try {
  const BetterSocialCore = require('../../services/bettersocial');
+ } catch (error) {
+   console.error('Failed to load BetterSocialCore:', error);
+   process.exit(1);
+ }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->